compiler/lib/src/main/scala/util/Error.scala

```diff
@@ -48,6 +48,9 @@ sealed trait Error {
       case SemanticError.DuplicateDictionaryName(kind, name, loc, prevLoc) =>
         Error.print (Some(loc)) (s"duplicate ${kind} name ${name}")
         printPrevLoc(prevLoc)
+      case SemanticError.DuplicateStateMachineInstance(name, loc, prevLoc) =>
+        Error.print (Some(loc)) (s"duplicate name ${name}")
```

Comment:
Let's say "duplicate state machine instance name" in the error message. Also, let's keep the case alphabetized.

Reply:
Fixed